### PR TITLE
🌐 Add French definition for "women and people of colour"

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hello!

This PR aims to add the French definition for **"women and people of colour"**. 

A link has been added to the existing English definition (and vice versa).

#### Notes

To the editor who would have time to go through this PR, I wasn't too sure about some specific terms but here are the resources I based my translations on:

* "personnes de couleur" & "non-white" for "people of colour" &  "non-white": https://www.linternaute.fr/dictionnaire/fr/definition/personne-de-couleur

Thanks in advance for the proofreading! 🌸

### CODE REVIEW

File: 11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+souvent utilisé comme une expression pour englober "qui n'est pas de couleur blanche, qui n'est pas un homme", cherchant à assurer la solidarité de ces deux groupes
+
+## Problèmes
+
+Qu'en est-il des femmes de couleur ? En tant que femme de couleur, je suis partagée à la fois entre les femmes et les personnes de couleur.
+
+## Impact
+
+De ce fait, cett expression suscite un sentiment d'effacement chez les femmes de couleur. Elle néglige également les
```
Comment (deammer) [inline 11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md]: Small typo: `cette`

**Comment by deammer** [comment]
Thank you for doing this @CharlesMangwa 🙌

File: 11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md (hunk 1 above)
Comment (CharlesMangwa) [reply 11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md]: All good now, thank you for your sharp eyes!

**Comment by CharlesMangwa** [comment]
Happy I'm able to help @deammer!

Just corrected the typo you found: I'm grateful for your time and feedback 🙏

File: 11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+defined: true
+excerpt: souvent utilisé comme une expression pour englober "qui n'est pas de couleur blanche, qui n'est pas un homme", cherchant à assurer la solidarité de ces deux groupes ; efface les femmes de couleur, les personnes non binaires
+speech: nom
+alt_words:
+  - personnes de couleur et femmes blanches
+  - personnes de couleur
+  - personnes blanches non-binaires, et femmes blanches
+  - trouver des moyens de recadrer les raisons de l'existence de cette dynamique
+  - ou omettre
+---
```
Comment (tatianamac) [inline 11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md:22]: Add references here with 
```suggestion
reading:
  - text: Name of article
    href: https://url
  - text: Name of article
    href: https://url
---
```

**Review by tatianamac — APPROVED** [review]
Suggestion to add references

**Comment by CharlesMangwa** [comment]
@tatianamac On top of the one I mentioned in the description, I added two other references I read while translating this definition. I'd say they're valuable testimonies which add more depth, explanation and understanding of the definition, its subject and context, which is why I put them in.

**Pull request merged**